src/plugins/gui/scenes/RewardsCardDashboardScene.tsx

```diff
@@ -100,15 +99,17 @@ const CardListContainer = styled(View)<{ bottomSpace: number }>(props => ({
 }))
 
 const CardListItem = styled(View)(props => ({
-  marginLeft: props.theme.rem(1)
+  backgroundColor: props.theme.modal,
```

Comment:
Would be nice if the cards adhered to industry standard CR80 ratio (3.375 inches wide by 2.125 inches high)